src/polygon/src/polygon.cpp

```diff
@@ -965,7 +963,7 @@ std::vector<glm::ivec3> TriangulateIdx(const PolygonsIdx &polys,
       triangles = triangulator.Triangulate();
       updatedPrecision = triangulator.GetPrecision();
     }
-#if MANIFOLD_EXCEPTION
+#ifdef MANIFOLD_EXCEPTIONS
```

Comment:
@pca006132 it seems we had a typo here that was keeping us from checking if our triangulation was valid. That's why we got the wrong number of triangles - it actually has a few CW triangles, which is what I was expecting to get caught first. Now I'm working on figuring out the triangulator error this uncovered.

Reply:
ah! interesting!